app/grandchallenge/workstation_configs/serializers.py

```diff
@@ -84,4 +86,12 @@ class Meta:
             "show_flip_tool",
             "show_window_level_tool",
             "show_reset_tool",
+            "enable_contrast_enhancement",
         ]
+
+        def get_enabled_preprocessors(self):
+            if self.enable_contrast_enhancement:
```

Comment:
The signature here is not correct, it should be `def get_enabled_preprocessors(self, obj):` (see https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield)

`self` here refers to the serializer, not the instance of the model, so the lookup you will need is then `obj.enable_contrast_enhancement`

Reply:
oh yes, thank you! Fixed it!